test/silicon_lda.jl

```diff
@@ -4,6 +4,13 @@ include("silicon_runners.jl")
     run_silicon_lda(Float64, Ecut=7, test_tol=0.03, n_ignored=0, grid_size=17, scf_tol=1e-5)
 end
 
+if run_expensive
```

Comment:
make that caps so it's more clear it's a global?

Reply:
also it's nice when we can run a specific test by itself by including the test directly, this breaks that

Reply:
Good point, I'll add a check whether the variable is defined.